src/frontend/src/components/authenticateBox.test.ts

```diff
@@ -1,10 +1,11 @@
+import { I18n } from "$src/i18n";
 import { html, render } from "lit-html";
 import { authnTemplates } from "./authenticateBox";
 
 test("anchors are forwarded", async () => {
   const addDevice: (anchor?: bigint) => void = jest.fn();
   const recover: (anchor?: bigint) => void = jest.fn();
-  const pages = authnTemplates({
+  const pages = authnTemplates(new I18n(), {
```

Comment:
note: you should use `$src/utils/i18n` (and not `$src/i18n`, confusing, I know...) to avoid the test picking using local storage, see [this](https://github.com/dfinity/internet-identity/blob/9107c5628dec66261ca1f5b8c683f8117bbde3da/src/frontend/src/flows/recovery/confirmSeedPhrase.test.ts#L6) for inspiration

Reply:
I had taken this from the `addDeviceSuccess.test.ts`